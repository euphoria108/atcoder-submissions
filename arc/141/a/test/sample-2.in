8
131313
131413
131314
131213
131312
131212
131412
10000
