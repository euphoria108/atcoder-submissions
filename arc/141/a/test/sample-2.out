131313
131313
131313
131131
131131
131131
131313
9999
